### PR TITLE
Keep producing xenial artifacts?

### PR DESCRIPTION
**Pull request by mikaelarguedas**
We mentioned it in the past but don't remember the final decision if we made one.

Should we keep producing xenial fat archives for now?

(slightly related to https://github.com/ros2/ci/issues/187)

### CODE REVIEW

**Comment by dhood** [comment]
this wasn't added in https://github.com/ros2/ci/pull/178 because I was under the impression we wanted one xenial job to rule them all (which would require some refactoring to let CI jobs test the bridge) but this approach is more straightforward :+1:

**Comment by dhood** [comment]
However, because it was out of scope, "bionic or xenial" support was not added for the packaging job in that PR, so there will be more work than just enabling this job (e.g. [it still hard-codes melodic](https://github.com/ros2/ci/blob/b0bf9856f073a1d52773a78601cbc50afc553f0d/job_templates/packaging_job.xml.em#L142)) so I don't think this will build as-is. Shall I work on a follow-up or have you started it already @mikaelarguedas ?

**Comment by mikaelarguedas** [comment]
Oh so the option was added to the packaging job parameters but the parameter is ignored.. That's confusing.

> Shall I work on a follow-up or have you started it already @mikaelarguedas ?

No I have not, and am out for the day, a follow-up will be appreciated :bowing_man:

File: create_jenkins_job.py [hunk 1]
```diff
@@ -252,6 +252,16 @@ def create_job(os_name, job_name, template_file, additional_dict):
...
         })
 
+        ubuntu_distro = 'xenial'
+        # configure a nightly xenial packaging job
+        job_name = 'packaging_{0}_{1}'.format(ubuntu_distro, os_name)
+        create_job(os_name, job_name, 'packaging_job.xml.em', {
+            'cmake_build_type': 'RelWithDebInfo',
+            'test_bridge_default': 'true',
+            'time_trigger_spec': PERIODIC_JOB_SPEC,
+            'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
```
Comment (dhood) [inline create_jenkins_job.py]: FYI, this accidentally left out passing the ubuntu distro through, which was fixed up in https://github.com/ros2/ci/commit/dce075c47ccad6e5fc397699e56f6c9e9cca36ce

**Comment by mikaelarguedas** [comment]
Thanks @dhood for the follow-up and fixups!

**Pull request merged**